komodo/symlink/suggester/cli.py

```diff
@@ -57,49 +61,64 @@ def _parse_args():
     parser.add_argument("--git-fork", help="git fork", default="equinor")
     parser.add_argument("--git-repo", help="git repo", default="komodo-releases")
     parser.add_argument("--git-ref", help="git ref", default="main")
+    parser.add_argument(
```

Comment:
Suggest adding `argparse.ArgumentDefaultsHelpFormatter` to this parser --- I added it to all of the other CLIs last week, but missed this one. (That other PR is not merged yet so I can also add it there, in which case I think I'll wait till this one is merged.)

Reply:
I can do it here!